lib/motions/open-file-under-cursor.coffee

```diff
@@ -0,0 +1,58 @@
+{Motion} = require './general-motions'
+path = require 'path'
+{Directory} = require 'atom'
+shell = require 'shell'
+{exec} = require 'child_process'
+utils = require '../utils'
+
+class OpenFileUnderCursor extends Motion
+  execute: (count) ->
+    ###
+      /usr/bin/test
+      ../../.travis.yml
+      ./search-motion
+      ./index
+      http://github.com
+      https://thing.com/item%20test
+      //github.com
+    ###
+    wordRegex = /[a-z0-9\.\-_\/\\%:]+/i
+    @editor.getCursors().forEach (cursor) =>
+      currentWord = utils.getCurrentWord @editor, cursor, wordRegex
+      # range = cursor.getCurrentWordBufferRange wordRegex: wordRegex
+      # selectedPath = @editor.getTextInRange range
+      range = currentWord.range
+      selectedPath = currentWord.word
```

Comment:
I wanted to be able to use selectedPath, but keep the `getCurrentWord` method as generic as possible, so i'd end up with an extra declaration anyways or I'd have to use `word` in the remaining code! Does that make sense?